README.md

```diff
@@ -128,8 +128,51 @@ Docker
 
 - [Developing minetestserver with Docker](doc/developing/docker.md)
 
-We provide a Dockerfile that can be used to build the server.
+We provide Minetest server Docker images using the GitHub container registry.
 
+Images are built on each commit and available using the following tag scheme:
+
+* `ghcr.io/minetest/minetest:master` (latest build)
+* `ghcr.io/minetest/minetest:<tag>` (specific tag)
+* `ghcr.io/minetest/minetest:latest` (latest tag)
```

Comment:
i dont see any way how "latest" isn't equivalent to "master" except maybe for a short time after a release
so is this worth documenting?

Reply:
`latest` is the latest git tag, `master` is the latest build from the master branch

https://github.com/docker/metadata-action?tab=readme-ov-file#basic

Reply:
it's not clear at all that "latest" is that latest stable version

Reply:
Better now?